package.json

```diff
@@ -1,14 +1,15 @@
 {
-  "name": "inspec-objects",
-  "version": "1.0.0",
-  "description": "Typescript objects for InSpec profiles",
-  "main": "index.ts",
+  "name": "@mitre/inspec-objects",
```

Comment:
Do we want to name the repository `inspec-objects` instead of `ts-inspec-objects`? Seems like a good idea for them to match.

Reply:
This was chosen so it wouldn't be confused with [inspec/inspec-objects](https://github.com/inspec/inspec-objects)

Reply:
Understood, but we also have this repo, which makes this unclear. 
https://github.com/mitre/inspec-objects